app/views/incidents/show.html.erb

```diff
@@ -24,7 +24,7 @@
   </p>
   <% warning_text = "" %>
   <% unless @incident.resolved_at %>
-    <% warning_text += "This incident is pending, a final ruling and public description may not be available.<br/>" %>
+    <% warning_text += "Note: This log has not been posted yet.<br/>" %>
```

Comment:
In my opinion, this text "hides" too much of what's going on in the background. Something like "has not been resolved yet, so it is not publicly visible" seems like a fair compromise

Reply:
Changed!